### PR TITLE
Add first docs

### PR DESCRIPTION
**Pull request by traversaro**
@prashanthr05 @triccyx can you check if they make sense? Thanks!

### CODE REVIEW

**Comment by traversaro** [comment]
Also  @lrapetti may be interested in this.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,72 @@
...
+
+# About versioning
+The project is undergoing _heavy_ development: APIs may still be subject to changes, and some functionalities may be broken. 
+
+
+# Background
+This repository contains plugins for the [Gazebo robotic simulator](http://gazebosim.org/) to enable
+[co-simulation](https://en.wikipedia.org/wiki/Co-simulation) between the physics simulation of Gazebo
+and simulation tools that support the [Functional Mock-up Interface (FMI)](https://fmi-standard.org/) standard.
+At the moment, the report just contains a plugin that **import** FMUs for simulation of the actuator dynamics.
```
Comment (prashanthr05) [inline README.md]: Functional Mock-up Unit (FMU)

File: plugins/actuator/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+| Parameter name | Type    | Description                 | Notes |
+|:--------------:|:-------:|:--------------------------: |:-----:|
+| name           | string  | Name of the actuator, used for printing debug and error messages. |  |
+| joint          | string  | Name of the joint. | The total list of joints contained in the model is scanned and the first joint that **ends** with this  name string is found. This is done to easily support nested models.  |
+| fmu            | string  | Filename of the FMU plugin to use for actuator co-simulation. | This name is passed to the [`gazebo::common::SystemPaths::FindFile`](http://osrf-distributions.s3.amazonaws.com/gazebo/api/9.0.0/classgazebo_1_1common_1_1SystemPaths.html#a9e03f07eac9f89d8c4c14af5660fa938) method to find the absolute location of the FMU file. Adding the directory containing the FMUs to the [`GAZEBO_RESOURCE_PATH`](http://gazebosim.org/tutorials?tut=components) should be sufficient to make it visible to the plugin. |
+| actuatorInputName | string | Name of the FMU input variable representing the actuator input. | This variable should be present in the FMU with causality INPUT. |
```
Comment (prashanthr05) [inline plugins/actuator/README.md]: If we could briefly introduce the definition of causality somwhere, it would be helpful.

File: README.md [hunk 3]
```diff
@@ -4,12 +4,11 @@
 
 # Overview
 - [About versioning](about-versioning)
```
Comment (prashanthr05) [inline README.md]: Link for about versioning is missing.

File: plugins/actuator/README.md (hunk 2 above)
Comment (traversaro) [reply plugins/actuator/README.md]: Addressed  in https://github.com/robotology-playground/gazebo-fmi/pull/1/commits/3472d3e6d6870a691a4c7ad2b4afed0f8e1bf4a2 . 
In general, if you have an FMU I suggest to use the [`fmpy info`](https://github.com/CATIA-Systems/FMPy) command to quickly check the causality of the variables of the FMU.

File: README.md (hunk 1 above)
Comment (traversaro) [reply README.md]: Addressed in https://github.com/robotology-playground/gazebo-fmi/pull/1/commits/1122817bad16f32951c2a51e680299295ee936a5 .

**Comment by traversaro** [comment]
@prashanthr05 please approve if you do not have any more  comment, thanks!

**Comment by prashanthr05** [comment]
Approved. Thanks.

File: README.md (hunk 3 above)
Comment (traversaro) [reply README.md]: Fixed  in https://github.com/robotology-playground/gazebo-fmi/pull/1/commits/1122817bad16f32951c2a51e680299295ee936a5 .

**Pull request merged**